hack/helm_vars/wire-server/values.yaml.gotmpl

```
@@ -278,6 +278,11 @@ galley:
           name: "cassandra-jks-keystore"
           key: "ca.crt"
     {{- end }}
+    postgresql:
+      host: "postgresql"
```

```suggestion
      host: postgresql
```

In YAML this should become a string anyways.